integration/confidential/lib.sh

```diff
@@ -204,7 +204,7 @@ configure_cc_containerd() {
 	sudo systemctl stop containerd
 	sleep 5
 	[ -n "$saved_containerd_conf_file" ] && \
-		cp -f "$containerd_conf_file" "$saved_containerd_conf_file"
+		sudo cp -f "$containerd_conf_file" "$saved_containerd_conf_file"
```

Comment:
Hi @ryansavino , double-checking it is really needed. Because `teardown_common()` in `integration/containerd/confidential/tests_common.sh` will restore the file (`mv -f "$SAVED_CONTAINERD_CONF_FILE" "/etc/containerd/config.toml"`) without sudo. I would expect it failing there too.

Reply:
i was getting errors in my test env from this line and the script was stopping. In the CI env, this file ends up being owned by root from some process:
```
root@amd-coco-ci-ubuntu2004-001:~# ls -ltar /etc/containerd/config.toml
-rw-r--r-- 1 root root 7066 Jun 26 15:02 /etc/containerd/config.toml
```